include/ck_tile/core/tensor/load_tile_transpose.hpp

```diff
@@ -15,6 +15,8 @@
 #include "ck_tile/core/numeric/math.hpp"
 #include "ck_tile/core/utility/type_traits.hpp"
 
+#define DS_READ_TR_SIZE 8
```

Comment:
should we pair it with #undef at the end of the header file so it doesn't leak outside?

Reply:
It is actually used outside. See https://github.com/ROCm/composable_kernel/pull/2436#discussion_r2203467169